docs/labs.md

```diff
@@ -7,6 +7,14 @@ to `Settings->Labs`. This list is non-exhaustive and subject to change, chat in
 **Be warned! Labs features are not finalised, they may be fragile, they may change, they may be
 dropped. Ask in the room if you are unclear about any details here.**
 
+## Matrix Spaces [MSC1772](https://github.com/matrix-org/matrix-doc/blob/matthew/msc1772/proposals/1772-groups-as-rooms.md) support (`feature_spaces`)
```

Comment:
I have been told in the past it is preferred to links to the PR overall for MSCs, since the rendered view might go away when the MSC is merged, etc.

```suggestion
## Matrix Spaces [MSC1772](https://github.com/matrix-org/matrix-doc/pull/1772) support (`feature_spaces`)
```